src/neon/how-tos/advance-analysis/undistort.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Undistorting Video and Gaze Data
+description: "Undistorting Neon's scene camera video and gaze data, using the intrinsic and extrinsic camera parameters."
+permalink: /neon/how-tos/advance-analysis/undistort/
+---
+# Correcting for the fish-eye lens distortion on Neon's scene camera
```

Comment:
I am not sure calling our lens a fish-eye lens is correct. Technically speaking I think we have "rectilinear" distortion. OpenCV has a separate model for calibrating fisheye cameras which we are not using. And finally, fish-eye distortion is usually pretty extreme, which is not the case for us.

So maybe we just call it "lens distortion" and drop the "fish-eye"?